scripts/do-release.sh

```diff
@@ -6,8 +6,17 @@ set -e
 . "${DAPPER_SOURCE}/scripts/lib/image_defs"
 # shellcheck source=scripts/lib/utils
 . "${DAPPER_SOURCE}/scripts/lib/utils"
+# shellcheck disable=SC1091
```

Comment:
```suggestion
# shellcheck source=scripts/lib/debug_functions
```

Does this work, instead of ignoring it?

Applies to similar elsewhere.

Reply:
@dfarrell07 It doesn't work because the source file should be existed in the project but the `debug_functions` is part of shipyard
see https://github.com/submariner-io/releases/runs/4408139119?check_suite_focus=true

Reply:
You should be able to drop this commit now as https://github.com/submariner-io/shipyard/pull/690 was merged